docs/manual/network.md

```diff
@@ -0,0 +1,156 @@
+```bash
+# ping srl1 management interface from srl2
+❯ docker exec -it clab-srl02-srl2 sr_cli "ping 172.20.20.3 network-instance mgmt"
+x -> $176x48
+Using network instance mgmt
+PING 172.20.20.3 (172.20.20.3) 56(84) bytes of data.
+64 bytes from 172.20.20.3: icmp_seq=1 ttl=64 time=2.43 ms
+```
+
+!!!note
+    If you run multiple labs without changing the default management settings, the containers of those labs will end up connecting to the same management network with their management interface.
+
+### configuring management network
+Most of the time there is no need to change the defaults for management network configuration, but sometimes it is needed. For example, it might be that the default network ranges are overlapping with existing addressing scheme on the lab host.
+
+For such cases the users need to add the `mgmt` container at the top level of their topology definition file:
+
+```yaml
+name: srl02
+
+mgmt:
+  network: custom_mgmt                # management network name
+  ipv4_subnet: 172.100.100.0/24       # ipv4 range
+  ipv6_subnet: 2001:172:100:100::/80  # ipv6 range (optional)
+
+topology:
+# the rest of the file is omitted for brevity
+```
+
+With this settings in place container will get their IP addresses from the specified ranges accordingly.
+
+### connection details
+When containerlab needs to create the management network it asks the docker daemon to do this. Docker will fullfil the request and will create a network with the underlying linux bridge interface backing it. The bridge interface name is generated by the docker daemon, but it is easy to find it:
+
+```bash
+# list existing docker networks
+# notice the presence of the `clab` network with a `bridge` driver
+❯ docker network ls
+NETWORK ID          NAME                DRIVER              SCOPE
+5d60b6ec8420        bridge              bridge              local
+d2169a14e334        clab                bridge              local
+58ec5037122a        host                host                local
+4c1491a09a1a        none                null                local
+
+# the underlying linux bridge interface name follows the `br-<first_12_chars_of_docker_network_id> pattern
+# to find the network ID use:
+❯ docker network inspect clab -f {{.ID}} | head -c 12
+d2169a14e334
+
+# now the name is known and its easy to show bridge state
+❯ brctl show br-d2169a14e334
+bridge name	        bridge id		    STP enabled	  interfaces
+br-d2169a14e334		8000.0242fe382b74	no		      vetha57b950
+							                          vethe9da10a
+```
+
+As explained in the beginning of this article, containers will connect to this docker network. This connection is carried out by the `veth` devices created and attached with one end to bridge interface in the lab host and the other end in the container namespace. This is illustrated by the bridge output above and the diagram above in the beginning of the article.
```

Comment:
`the diagram above in the beginning of the article.`
the diagram at the beginning the of the article